### PR TITLE
link client logos on start page on hover

### PR DESCRIPTION
**Pull request by marcoow**
bildschirmfoto 2017-02-17 um 15 01 09

### CODE REVIEW

File: css/typography.scss [hunk 1]
```diff
@@ -116,6 +116,13 @@ h5 {
   }
 }
 
+#main-section.index {
+  .client-logos {
+    a {
+    }
+  }
+}
+
```
Comment (gr0uch) [inline css/typography.scss]: There doesn't seem to be any rules in here?

Comment (marcoow) [reply css/typography.scss]: argh, right - removed it

File: css/app.scss [hunk 2]
```diff
@@ -233,18 +237,48 @@ ul {
...
+    a {
+      position: absolute;
+      display: block;
+      width: 100%;
+      top: calc(50% - 1em - 50px);
+      z-index: 9999;
+      padding: 50px;
+      opacity: 0;
+      -webkit-transition: opacity 0.15s ease-in-out;
+      -moz-transition: opacity 0.15s ease-in-out;
```
Comment (gr0uch) [inline css/app.scss]: I think these haven't been needed since 2012-2013: http://caniuse.com/#feat=css-transitions

Comment (marcoow) [reply css/app.scss]: Oh, cool - wasn't aware. I'll remove those in a separate PR.

**Review by gr0uch — APPROVED** [review]

**Pull request merged**